pavelib/assets.py

```diff
@@ -641,14 +641,28 @@ def restart_django_servers():
     ))
 
 
-def collect_assets(systems, settings):
+def collect_assets(systems, settings, debug=None, collectstatic_log=None):
     """
     Collect static assets, including Django pipeline processing.
     `systems` is a list of systems (e.g. 'lms' or 'studio' or both)
     `settings` is the Django settings module to use.
+    `debug` and `collectstatic_log` are used for determining where to
+       pipe collectstatic logging.
     """
+
+    # unless specified, collectstatic (which can be very verbose) pipes to /dev/null
+    collectstatic_stdout_str = "> /dev/null"
+    if debug:
+        # pipe to console
+        collectstatic_stdout_str = ""
+    if collectstatic_log:
+        # pipe to specified file
+        collectstatic_stdout_str = "> {output_file}".format(output_file=collectstatic_log)
```

Comment:
I think based on the number of callers (there's only one where you're actually passing in the debug flag, from what I can see), I'd still lean towards the refactor into a single argument that I proposed above.